promoter/src/main/java/ch/bfh/zodnrk/promoter/service/impl/DefaultPromoterService.java

```diff
@@ -0,0 +1,41 @@
+package ch.bfh.zodnrk.promoter.service.impl;
+
+import ch.bfh.zodnrk.camp.model.Party;
+import ch.bfh.zodnrk.promoter.client.ArenaClient;
+import ch.bfh.zodnrk.promoter.client.CampClient;
+import ch.bfh.zodnrk.promoter.service.PromoterService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class DefaultPromoterService implements PromoterService {
+
+    private static final Logger LOG = LoggerFactory.getLogger(DefaultPromoterService.class);
+
+    @Autowired
+    private CampClient campClient;

```

Comment:
I know, it was provided, but for testing, dependency injection would be better. But no need to change...